lib/autoproj/daemon/git_api/pull_request.rb

```diff
@@ -85,6 +96,23 @@ def draft?
                 def mergeable?
                     @model["mergeable"]
                 end
+
+                # Resolve the list of recursive dependencies for this pull request
+                #
+                # @return [Set<GitAPI::PullRequest>]
+                def recursive_dependencies
```

Comment:
How is this recursive?

Reply:
It resolves the 'recursive dependencies'. Which means it resolves the dependencies of the dependencies. Does not mean the algorithm to do so has to be recursive.